src/screens/NewNote/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { API } from "aws-amplify";
+import React, { Component } from "react";
+import { ControlLabel, FormControl, FormGroup } from 'react-bootstrap';
+import LoaderButton from '../../components/LoaderButton';
+import config from '../../config';
+import { s3Upload } from '../../libs/awsLib';
+import './index.css';
+
+class NewNote extends Component {
+  constructor(props) {
+    super(props);
+
+    this.file = null;
+
+    this.state = {
+      isLoading: null,
+      content: "",
+    };
+  }
+
+  saveNote(body) {
+    return API.post("notes", `/notes`, { body });
+  }
+
+  validateForm = () => (this.state.content.length)
+
+  formatFilename(str) {
+    return str.replace(/^\w+-/, "");
```

Comment:
What is this regrex used for? You can define a constant to make it more readable